src/Ref.ts

```diff
@@ -117,7 +135,7 @@ export function getEmptyRefAt(document: vscode.TextDocument, position: vscode.Po
   let searchRange = new vscode.Range(s, position);
   let precedingChars = document.getText(searchRange);
 
-  if (precedingChars == '[[') {
+  if (precedingChars == '[[' || precedingChars == '](') {
```

Comment:
This part of the code I think is only necessary if we are doing auto-completion behavior for HyperLinks -- are we changing completion, or just backlinks? If changing only backlinks, I'm not sure we need this here (and it might cause an unexpected behavior change where people immediately start getting completions from another provider after typeing `](`

Reply:
You're absolutely right; this is not needed. This is still some residual code that I forgot to remove.